src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const prop in robot) {
```

Comment:
Consider using more meaningful variable name instead of 'prop', such as 'key' or 'property'.